### PR TITLE
Gradle intro: Issue #75 corrected

### PR DESCRIPTION
**Pull request by mtavi**

### CODE REVIEW

File: gradle_intro/README.md [hunk 1]
```diff
@@ -10,11 +10,12 @@ The goal of this exercise is to successfully download the latest version of a de
...
 
 1. Navigate to your exercise folder
-1. Add your gradle repository to `build.gradle`
+1. Add your gradle repository to `build.gradle` Go to Artifactory and navigate to your $KATA_USERNAME-generic-gradle-1 repository.
++    Copy URL to file path from the 'General' page
 
     ```groovy
     repositories {
-        maven { url "http://ArtifactoryUrl.com/artifactory/$KATA_USERNAME-generic-gradle-1" }
+        maven { url "<Copied from your Artifactory $KATA_USERNAME-generic-gradle-1 repository General page>" }
```
Comment (sofusalbertsen) [inline gradle_intro/README.md:18]: I think this part is hard to follow, as I (as a student) am uncertain if all text in <> should be exchanged or not.
Possible solution:
Only write `copied Artifactory URL` in the <> section

**Review by sofusalbertsen — CHANGES_REQUESTED** [review]

File: gradle_intro/README.md (hunk 1 above)
Comment (mtavi) [reply gradle_intro/README.md:18]: Hmm, it is not Artifactory URL (https://artifactoryurl.com/artifactory) that is needed here, but the repo link.
Maybe
`maven { url "<Copied from your Artifactory repository General page>" } `
would convey the meaning better?

Comment (sofusalbertsen) [reply gradle_intro/README.md]: I think so too yes :)

**Review by sofusalbertsen — APPROVED** [review]
